std_msgs/Header header                  # time stamp
geometry_msgs/Vector3 gyro              # Gyro reading - In degrees/sec
geometry_msgs/Vector3 linear_acc        # Accelerometer reading -  Acceleration in m/sec^2
float32[4] q                            # Quaternion reading [w, x, y, z]